### PR TITLE
Feature/schema

### PR DESCRIPTION
**Pull request by danalvrz**
- Created schema_based_on_diagram.sql 
- Built database from the diagram
- Created  a "join table" for many-to-many relationships
- Added the FK indexes

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    treatment_id INT,
+    CONSTRAINT medical_history_id_fk FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+    CONSTRAINT treatment_id_fk FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+
+
+CREATE INDEX medical_history_id_index ON invoices (medical_history_id asc);
+CREATE INDEX invoice_id_index ON invoice_items (invoice_id asc);
+CREATE INDEX medical_history_treatment_index ON medical_treatments (medical_history_id asc);
+CREATE INDEX treatment_id_index ON medical_treatments (treatment_id asc);
```
Comment (julie-ify) [inline schema_based_on_diagram.sql]: - [x] Kindly create all the `indexes` of all the foreign keys. You have only 4 indexes created remaining 2. The remaining indexes are `INDEX ON medical_histories (patient_id);` and `INDEX ON invoice_items (treatment_id);`

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @danalvrz, @rtonata88, and @oliverSCZ,

Good job so far!

## Highlights!
- Good feature branch name ✔️  
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (danalvrz) [reply schema_based_on_diagram.sql]: done, thanks!

**Review by Lordkaito — APPROVED** [review]
Hi @danalvrz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**